.github/workflows/regenerate-stateful-test-disks.yml

```diff
@@ -0,0 +1,76 @@
+name: Regenerate test state
+
+on:
+  workflow_dispatch:
+
+env:
+  PROJECT_ID: zealous-zebra
+  ZONE: europe-west1-b
```

Comment:
FYI: this was also the best area for Tor relays

Reply:
Interesting! Might be a good crossroads of the internet 😁

Reply:
It definitely is - good US East + European connectivity.